marklogic-data-hub-central/ui/src/components/explore/entity-type-display-settings-modal/entity-type-display-settings-modal.tsx

```diff
@@ -0,0 +1,198 @@
+          </div>
+        </div>);
+      }
+    },
+    {
+      text: getHeaderLabel("Icon", entityTypeDisplaySettings.icon),
+      width: "10%",
+      align: "center" as "center",
+      headerAlign: "center",
+      formatter: (text, row) => {
+        let entityInfo = graphConfig.sampleMetadata["modeling"]["entities"][row.entityType];
+        let iconsrc = entityInfo && entityInfo.icon ? entityInfo.icon : entityIcon;
+        return <span className={styles.valueContainer}><img src={iconsrc} aria-label={`${row.entityType}-icon`}/>
+          <span className={styles.editIconContainer}><FontAwesomeIcon icon={faPencilAlt} size="sm" onClick={(e) => { e.preventDefault(); }} className={styles.editIcon} aria-label={`edit-${row.entityType}-icon`} /></span>
+        </span>;
+      }
+    },
+    {
+      text: getHeaderLabel("Entity Label", entityTypeDisplaySettings.entityLabel),
+      width: "25%",
+      formatter: (text, row) => {
+        return (
+          <Select
+            value={entityLabels[row.entityType]}
+            options={renderOptions(row.entityType)}
+            onChange={(e) => onEntityLabelChange(row, e)}
+            className={styles.columnData}
+            classNamePrefix="select"
+            aria-label={`${row.entityType}-label-select-dropdown`}
+          />);
+      },
+    },
+    {
+      text: getHeaderLabel("Properties on Hover", entityTypeDisplaySettings.propertiesOnHover),
+      width: "35%",
+      formatter: (text, row) => {
+        return (
+          <Select
+            value={propertiesOnHover[row.entityType]}
+            isMulti
+            options={renderOptions(row.entityType)}
+            className={styles.columnData}
+            classNamePrefix="select"
+            aria-label={`${row.entityType}-propertiesOnHover`}
+            onChange={(e) => onPropertiesOnHoverChange(row, e)}
+          />
+        );
+      }
+    }
+  ];
+
+  const modalFooter = (
+    <div className={styles.editFooter}>
+      <div
+        className={styles.footer}
+      >
+        <HCButton
+          size="sm"
+          variant="outline-light"
+          aria-label={`cancel-multiple-ruleset`}
+          onClick={closeModal}
+        >Cancel</HCButton>
+        <HCButton
+          className={styles.saveButton}
+          size="sm"
+          aria-label={`confirm-multiple-ruleset`}
+          variant="primary"
+        >Save</HCButton>
+      </div>
+    </div>
+  );
+
+
+
+
+  return (
+    <Modal
+      show={props.isVisible}
+      dialogClassName={styles.modal1400w}
+    >
+      <Modal.Header className={"bb-none align-items-start"}>
+        <span className={"fs-4"}>
+          Entity Type Display Settings
+        </span>
+        <button type="button" className="btn-close" aria-label="Close" onClick={closeModal}></button>
+      </Modal.Header>
+      <Modal.Body>
+        <div id="entityTypeDisplaySettingsContainer" data-testid="entityTypeDisplaySettingsContainer">
```

Comment:
What is the exptected whene there are no entities created yet?

Reply:
When there are no entities created yet, then the display settings modal option should be disabled, with helpful tooltip information for the users.